spec/system/admin_branding_spec.rb

```diff
@@ -0,0 +1,212 @@
+# frozen_string_literal: true
+
+describe "Admin Branding Page", type: :system do
+  fab!(:admin)
+  fab!(:image_upload)
+
+  let(:branding_page) { PageObjects::Pages::AdminBranding.new }
+  let(:admin_branding_logo_form) { PageObjects::Components::AdminBrandingLogoForm.new }
```

Comment:
Does it make sense to make `admin_branding_logo_form` a method on `PageObjects::Pages::AdminBranding` page object given that the component is rendered within the page? We could shorten the method name so it becomes `branding_page.logo_form`.